README.md

```diff
@@ -43,6 +43,8 @@ Marcel::MimeType.for Pathname.new("example.png"), name: "example.ai"
 # As "application/illustrator" is not a more specific type of "image/png", the filename is ignored
 ```
 
+`Marcel::MimeType.for` is Marcel's only public interface.
```

Comment:
`Marcel::MimeType.extend` and `Marcel::Magic` are all public API as well, so that sentence isn't right. The gist is that `Marcel::MimeType.for` is how the gem is used to identify mimes, so we can compromise on that. Let's amend the the beginning sentence to: "Marcel attempts to choose the most appropriate content type for a given file by looking at the binary data, the filename, and any declared type (perhaps passed as a request header). This is done via the `Marcel::MimeType.for` method, and is used like this:"

Reply:
Sounds good to me. Changes pushed.